recipes/gis-metadata-parser/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "gis-metadata-parser" %}
+{% set version = "2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gis-metadata-parser-{{ version }}.tar.gz
+  sha256: 5008cab4d40667d5248b2aba4db9b416f50c1139045c9b2faadd2c199cb201a4
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - poetry
+    - python >=3.6,<4.0
+  run:
+    - frozendict >=2.0,<3.0
+    - parserutils >=2.0.1,<3.0.0
+    - python >=3.6,<4.0
```

Comment:
Is there some reason you're adding upper bounds that [upstream doesn't have](https://github.com/consbio/gis-metadata-parser)?

Reply:
Only that these bounds  came direct from [castle grayskull](https://github.com/conda-incubator/grayskull). I am more than happy to pull them.

Reply:
I think that these are also actually being parsed out of the [poetry.lock file](https://github.com/consbio/gis-metadata-parser/blob/main/poetry.lock#L30) by gray skull. For example, the parser-utils dep there specifies python >=3.6 <4.0. I think grayskull is correct in how it specified these deps based on the lock file.

Reply:
Ah, they came from translating the `~=` in `pyproject.toml`. Fair enough.